### PR TITLE
Master move repo delete state fragments

### PR DESCRIPTION
**Pull request by claudiasima-ro**

### CODE REVIEW

**Review by georgiana-ecobici — APPROVED** [review]

File: plugins/org.polarsys.capella.scenario.editor.embeddededitor/src/org/polarsys/capella/scenario/editor/embeddededitor/commands/XtextToDiagramCommands.java [hunk 1]
```diff
@@ -827,6 +884,23 @@ private static boolean isSameMessage(EObject m, SequenceMessage seqMessage) {
     }
     return false;
   }
+  
+
+  private static boolean isSameStateFragment(Scenario scenario, EObject fragment, TimeLapse timelapse) {
```
Comment (MalinaStefaniaStoicanescu) [inline plugins/org.polarsys.capella.scenario.editor.embeddededitor/src/org/polarsys/capella/scenario/editor/embeddededitor/commands/XtextToDiagramCommands.java:889]: you could remove scenario from parameters, it seems that it's never used

File: plugins/org.polarsys.capella.scenario.editor.embeddededitor/src/org/polarsys/capella/scenario/editor/embeddededitor/commands/XtextToDiagramCommands.java [hunk 2]
```diff
@@ -788,6 +832,19 @@ private static boolean foundCapellaMessageInXText(SequenceMessage capellaSequenc
     return false;
   }
 
+  
+  private static boolean foundCapellaStateFragmentInXText(Scenario scenario, TimeLapse timelapse,
```
Comment (MalinaStefaniaStoicanescu) [inline plugins/org.polarsys.capella.scenario.editor.embeddededitor/src/org/polarsys/capella/scenario/editor/embeddededitor/commands/XtextToDiagramCommands.java:836]: if you remove scenario from isSameStateFragment, you could remove it from this method too

File: plugins/org.polarsys.capella.scenario.editor/src/org/polarsys/capella/scenario/editor/helper/EmbeddedEditorInstanceHelper.java [hunk 3]
```diff
@@ -395,4 +396,24 @@ public static String getModelContent() {
     return ScenarioExt.getAvailableFunctionsStateFragment(element.getRepresentedInstance()).stream()
         .collect(Collectors.toList());
   }
+  
+  public static String getStateFragmentType(StateFragment capellaStateFragment) {
+    if (capellaStateFragment.getRelatedAbstractFunction() != null) {
+      return DslConstants.FUNCTION;
+    } else if (capellaStateFragment.getRelatedAbstractState() != null
+        && capellaStateFragment.getRelatedAbstractState() instanceof State) {
+      return DslConstants.STATE;
```
Comment (MalinaStefaniaStoicanescu) [inline plugins/org.polarsys.capella.scenario.editor/src/org/polarsys/capella/scenario/editor/helper/EmbeddedEditorInstanceHelper.java:405]: I remember that Mode extends State. in this case will return "state", even if it's mode. you could check if is instanceof Mode and return DslConstants.MODE and else return state 
also you could remove all 'else'-s, they are not necessary

Comment (claudiasima-ro) [reply plugins/org.polarsys.capella.scenario.editor/src/org/polarsys/capella/scenario/editor/helper/EmbeddedEditorInstanceHelper.java:405]: Done.

File: plugins/org.polarsys.capella.scenario.editor.embeddededitor/src/org/polarsys/capella/scenario/editor/embeddededitor/commands/XtextToDiagramCommands.java (hunk 2 above)
Comment (claudiasima-ro) [reply plugins/org.polarsys.capella.scenario.editor.embeddededitor/src/org/polarsys/capella/scenario/editor/embeddededitor/commands/XtextToDiagramCommands.java:836]: Done.

File: plugins/org.polarsys.capella.scenario.editor.embeddededitor/src/org/polarsys/capella/scenario/editor/embeddededitor/commands/XtextToDiagramCommands.java (hunk 1 above)
Comment (claudiasima-ro) [reply plugins/org.polarsys.capella.scenario.editor.embeddededitor/src/org/polarsys/capella/scenario/editor/embeddededitor/commands/XtextToDiagramCommands.java:889]: Done.

**Pull request merged**